h/static/styles/annotation-share-dialog.scss

```diff
@@ -0,0 +1,130 @@
+.share-dialog-wrapper.open .annotation-share-dialog {
+  display: flex;
+  flex-direction: column;
+}
+
+.annotation-share-dialog {
+  display: none;
```

Comment:
Something to think about here is how we ensure visual consistency between this and any other dialogs that we might have in future. For example, should the styling of the dialog outline here match the styling of the groups dropdown and sign-in dropdown?